.github/workflows/smartmodule-publish.yml

```diff
@@ -50,8 +48,8 @@ jobs:
         run: fluvio install smdk
       - name: Fluvio Login
         run: |
-          fluvio cloud login --email ${{ inputs.target_prod && secrets.HUB_USER_EMAIL || secrets.DEV_HUB_USER_EMAIL }} \
-          --password ${{ inputs.target_prod && secrets.HUB_USER_PASSWORD || secrets.DEV_HUB_USER_PASSWORD }} \
+          fluvio cloud login --email ${{ secrets.ORG_HUB_SA_NAME }} \
+          --password ${{ !inputs.target_prod && secrets.ORG_HUB_SA_PASSWD_DEV || secrets.ORG_HUB_SA_PASSWD_PROD }} \
```

Comment:
The passwords and PROD and DEV are swapped.  Looks good otherwise

Reply:
should be ok because of the `!`

Reply:
Oh wait, why invert inputs.targets_prod?